frontend/packages/core/src/AppProvider/workflow.tsx

```diff
@@ -30,6 +30,7 @@ interface BaseWorkflowConfiguration {
   group: string;
   /**
    * The path (subordinate to the group) where the workflow will exist.
+   * (optionally) use "" to override the landing page
```

Comment:
do we also want to add this comment [here](https://github.com/lyft/clutch/blob/main/frontend/packages/core/src/AppProvider/workflow.tsx#L72)?

i noticed that with the echo workflow, which is structured like
```
    path: "examples",
    group: "Examples",
    displayName: "Echo",
    routes: {
      echo: {
        path: "echo",
        component: EchoWorkflow,
        displayName: "Demo Workflow",
        description: "Workflow for demonstration purposes.",
      },
    },
```
both paths had to be `""` in order for the workflow to show up as the landing page

Reply:
Ah yeah, I had mine set as "/" so I forgot about it, let me add another comment